packages/core/src/mesh/PrimitiveMesh.ts

```diff
@@ -6,11 +6,15 @@ import { IndexFormat } from "../graphic/enums/IndexFormat";
 import { VertexElementFormat } from "../graphic/enums/VertexElementFormat";
 import { Mesh } from "../graphic/Mesh";
 import { VertexElement } from "../graphic/VertexElement";
+import { GLCapabilityType } from "../base/Constant";
 
 /**
  * Used to generate common primitve meshes.
  */
 export class PrimitiveMesh {
+  /** The max number of indices that Uint16Array can support. */
+  private static _indiceLimit: number = 65535;
```

Comment:
this limit count is not index count, is vertex limit Count. Use _uint16VertexLimit or _uint16MaxValue

Reply:
> this limit count is not index count, is vertex limit Count. Use _uint16VertexLimit or _uint16MaxValue

done